src/Insightly/InsightlyClient.php

```diff
@@ -304,4 +305,12 @@ public function updateOrganisation(Organisation $organisation)
     {
         return GetOrganisationResult::parseToResult($this->request(RequestInterface::PUT, 'Organisations/', null, json_encode($organisation->toInsightly())));
     }
+
+    /**
+     * {@inheritdoc}
+     */
```

Comment:
I noticed you added an `@inheritdoc` annotation here, probably to stay consistent with the others in this file. I've gone ahead and just removed all `@inheritdoc` annotations

Reply:
Yeah, I did not wanted to go down the rabbit hole of updating code styling until the PHP-CS-Fixer was added.